packages/blaze/builtins.js

```diff
@@ -226,6 +226,11 @@ Blaze.Each = function (argFunc, contentFunc, elseFunc) {
         });
       },
       changedAt: function (id, newItem, oldItem, index) {
```

Comment:
Could you please make an example showing what you observed?

We need tests for these changes anyway.